assets/stylesheets/commons/_slider.css

```diff
@@ -0,0 +1,91 @@
+/*----------------------------- Slider Style ----------------------------------*/
+#fwslider{
+	position: relative;
+	width: 100%;
+}
+
+#fwslider .slider{
+	display: none;
+	position: absolute;
+	top: 0;
+	left: 0;
+	z-index: 0;
+	width: 100%;
+}
+
+#fwslider .slider img{
+	width: 100%;
+}
+
+#fwslider .sliderPrev{
+	position: absolute;
+	top: 50%;
+	left: -50px;
+	z-index: 10;
+}
+
+#fwslider .sliderPrev span{
```

Comment:
Class name shouldn't use camel case as `sliderPrev`. Use slider-prev instead of.